src/components/common/forms/SignUpForm.tsx

```diff
@@ -0,0 +1,134 @@
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import { ErrorMessage } from "@hookform/error-message";
+import { useState } from "react";
+import { Link } from "react-router-dom"
+import Input from "@common/Input"
+import Button from "@common/Button"
+import Select from "@common/Select"
```

Comment:
Have you thought about having an index.tsx in the `common` folder root that exports all the components of the folder? Then you could do something like`import {Input, Button, Select} from "@common"`